Backend/CarRentalApp/CarRentalApp/appsettings.json

```diff
@@ -5,5 +5,26 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
-}
+  "AccessJwtConfig": {
+    "GenerationParameters": {
+      "Issuer": "loginServer",
+      "Audience": "clientsApp",
+      "LifeTimeSeconds": "300",
```

Comment:
Why exactly this value?

Reply:
Lifetime needs to be as short as possible to protect from token robbery. So if Bob steals Alice`s token, he will get only 5 min of access.